install.sh

```diff
@@ -330,21 +316,7 @@ do_install() {
 		;;
 
 		debian|raspbian)
-			dist_version="$(sed 's/\/.*//' /etc/debian_version | sed 's/\..*//')"
-			case "$dist_version" in
-				11)
-					dist_version="bullseye"
-				;;
-				10)
-					dist_version="buster"
-				;;
-				9)
-					dist_version="stretch"
-				;;
-				8)
-					dist_version="jessie"
-				;;
-			esac
+			dist_version="$(sed -n 's/^VERSION_CODENAME=//p' < /etc/os-release )"
```

Comment:
This could take the same approach as `get_distribution()`; https://github.com/docker/docker-install/blob/0225270d2de23f66c4f551d97e74e9f831adfdc8/install.sh#L163-L172

```bash
if [ -r /etc/os-release ]; then
	lsb_dist="$(. /etc/os-release && echo "$VERSION_CODENAME")"
fi
```

However, I'm not sure how reliable codename is here, taking `sid` into account (on Debian); not sure if `raspbian` also uses `sid`, but if so, then it's likely that the codename doesn't always match the version.

Reply:
That is correct. Sid does not set the VERSION_CODENAME in /etc/os-release. 
raspbian uses the codename of the next release "bookworm" or "testing" instead of sid. 
However these versions are not present at download.docker.com which would let the script fail at that point. (or is there a testing/unstable repo?)


I would mimic the "lsb_dist=" style with the debian dist_version. lsb_dist would be for getting the os type while dist_version would get the debian/raspbian codename as ubuntu still behaves drifferent.

```suggestion
			dist_version="$(. /etc/os-release && echo "$VERSION_CODENAME")"
```

Would that be alright?